articles/quickstart/spa/angular2/01-login.md

```diff
@@ -1,10 +1,11 @@
 ---
 title: Login
-description: This tutorial demonstrates how to add user login to an Angular (versions 2 and above) application using Auth0.
+description: This tutorial demonstrates how to add user login to an Angular application using Auth0.
 budicon: 448
 topics:
   - quickstarts
   - spa
+  - angular
   - angular2
```

Comment:
Why not remove this topic as well?

Reply:
🤔 it's a good question that I don't have a good answer for. I wasn't sure if there would be any negative effects of removing it, so decided to leave it. It will disappear very shortly anyway when the quickstart is replaced with the new stuff.